packages/web/src/stores/choices.ts

```diff
@@ -38,7 +38,7 @@ export const choicesStore = defineStore('choices', () => {
     if (params) {
       for (const key in params) {
         const reg = new RegExp(`{${key}}`, 'g')
-        translated = translated.replace(reg, params[key])
+        translated = translated.replace(reg, params[key] ?? '')
```

Comment:
note: the previous way this function worked was a bit in purpose, to display the dictionnary code itself (f.i. `program.something`) in case of an error. The idea was -  at least during developments - to display something missing and to fix, instead of hiding it